deployer/template/aws_cloudfront_distribution.go

```diff
@@ -0,0 +1,20 @@
+package template
+
+import (
+	"github.com/awslabs/goformation/cloudformation"
+	"github.com/awslabs/goformation/cloudformation/resources"
+)
+
+// AWS::CloudFront::Distribution
+
+func ValidateAWSCloudFrontDistribution(
```

Comment:
Do we want to add restrictions here on:
- HttpVersion restricted to http2
- IPV6Enabled set to true
- ViewerCertificate restrictions based on tags

We might also want some restrictions on the origins, but I don't think we have a good story on what those will look like yet